flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamCalc.scala

```diff
@@ -45,7 +45,7 @@ class DataStreamCalc(
     ruleDescription: String)
   extends Calc(cluster, traitSet, input, calcProgram)
   with CommonCalc
-  with DataStreamRel {
+  with DataStreamRel[CRow] {
```

Comment:
`DataStreamRel` does not need a type parameter, IMO. All `DataStreamRel` nodes should produce `CRow`.